src/pyird/image/wavcal.py

```diff
@@ -1,11 +1,85 @@
+from pyird.io.read_linelist import read_linelist
 from astropy.io import fits
 import numpy as np
 from scipy.signal import medfilt
+from scipy.optimize import leastsq
 import pandas as pd
 
+import matplotlib.pyplot as plt
+
+def pdat2matrix(pdat):
```

Comment:
Can you leave the google/napoleon type comment for all of the functions?
like [this one](https://github.com/prvjapan/pyird/blob/cc9dbbeeda312e67e0f6f76f33f520fbb042b97e/src/pyird/spec/rsdmat.py#L6)

These comments enable us to generate the API document automatically, in `documents`.

Reply:
The API example is [here](http://secondearths.sakura.ne.jp/pyird/pyird/pyird.html).

Reply:
added the comment for all of the functions. please check it out.

Reply:
Thanks. 
```python
Return:
```
should be 
```python
Returns:
```
probably....